Create Ground_station.tex (#67)
Capstone-Skynet/Capstone-Skynet.github.io: docs/design/Ground_station.tex

\textbf{Autopilot and ground station}

\subsection{Overview}

> The multirotor is going to be controlled via a commercial off the shelf flight controller unit called an autopilot. The autopilot will connect to a ground station running an open source software called APM Planner 2.0. These hardware and software packages provide the 

Reviewer: Present tense + active voice:

```suggestion
An autopilot is a commercial off-the-shelf flight controller that controls the multirotor. This project uses a well-established \textit{ArduPilot Mega (APM) 2.8} autopilot. The autopilot connects to a ground station computer running \textit{APM Planner 2.0}\cite{"insert link to APM Planner 2.0"}. Note that this is a separate software used only to configure the autopilot and multirotor controls, and should not be confused with the ground station software that we developed.
``` 

Also not finished?

\subsection{Features list}

\begin{enumerate}

    \item Real time monitoring of telemetry information
    \item Real-time control of the Multirotor
    \item Primary Flight Display (HUD)
    \item Simple commands to the Multirotor such as hover
    \item Waypoints that can be used to set a path for the drone to fly
    \item Real-time battery monitoring
    
\end{enumerate}

\underline{Non-exhaustive list of Flight modes}

\begin{enumerate}

    \item Stabilize: Self-levels the roll and pitch axis
    \item Alt hold: Holds altitude and self-levels the roll and pitch axis
    \item Loiter: Holds altitude and position using GPS
    \item RTL: Returns to the takeoff position
    \item Stabilize: Self-levels the roll and pitch axis
    \item Auto: Executes pre-defined mission using waypoints
    
\end{enumerate}

\subsection{Configuraton}

Configuring the autopilot system requires 1 hardware device and 2 software packages. The hardware is the autopilot, the physical device that will actually be mounted on the Multirotor. The software packages are the firmware, the software that will be uploaded onto the autopilot, and the ground station software.

\underline{Ground station software}

> The ground station software is the APM Planner 2.0. The software is easily installed via an installer that can be downloaded from the Ardupilot website.

Reviewer: Provide website?

\underline{Firmware}

The autopilot hardware we are using is the APM 2.8 which is an older autopilot that is no longer supported. To utilize the older autopilot we need to use an older firmware package available on the Ardupilot website in the archive section. A copy of the firmware will also be provided in the deliverables of the project. Installation of the firmware is done through ground station software.
To install the firmware.

\begin{enumerate}

    \item The relevant firmware will be provided as a deliverable in the project but can also be found on the archive section of the ardupilot website
    \item Plug in the ardupilot and open mission planner
    \item Select the relevant COM port and set the baud rate to 115200, do not press connect
    \item Click on ‘initial setup’ and then ‘install firmware’
    \item Click on the copter icon and then when prompted install the provided firmware.
    \item Click on ‘Connect’
    
\end{enumerate}

\subsection{Pre-arm safety checks}

Before every flight the computer goes through a series of automatic safety checks. If any of the checks fail then multirotor will refuse to fly. The checks are as follows:

\begin{enumerate}

>     \item That the radio has been calibrated

Reviewer: Active voice or a list of checklist using imperative verbs:

```suggestion
    \item Calibrate radio
```

or

```suggestion
    \item Radio calibration: set and check.
```

\textbf{Autopilot and ground station}

\subsection{Overview}

> In order for the multirotor to be controllable by a human being with minimal training, much of the flight systems need to be automated. We have decided for our project to use an open source autopilot called ‘Ardupilot’. The autopilot system consists of several hardware components and 2 software components which are split between the multirotor and the ground station.

Reviewer: ```suggestion
Flight control must be automated for the multirotor to be controllable by a user with minimal training. We use ArduPilot, an open source autopilot stack. The autopilot system consists of several hardware components and two software components which are split between the multirotor and the ground station.
```

\underline{Multirotor hardware}\

\begin{enumerate}

    \item Autopilot flight computer: The autopilot flight computer is the primary hardware component of the autopilot system on the multirotor. In the Ardupilot literature this device is referred to only as the ‘Autopilot’ but to avoid confusion with the broader autopilot system we will be referring to it as the ‘Autopilot flight computer’. The specific board that we are using is the APM 2.6. This board an older model and will thus be using legacy code but will have all of the required functionality.
  

>   \item GPS and Compass: The GPS and compass will allow the ground station to track the multirotor while it’s flying. Also enables the use of the automatic flight modes.

Reviewer: ```suggestion
  \item GPS and Compass: The multirotor navigates and self-aligns using the GPS and the compass in automatic flight modes.
```

>  \item Telemetry module: Radio transceiver that allows for communication with the ground.

Reviewer: Note: our set up don't have a telemetry module
Note: we should clarify that this is different that the radio transmission that controls the drone (and the autopilot). Telemetry is typically used for information like GPS tags, attitude and altitude indicators, etc.

Author: I know, but on the ardupilot website it was referred to as the telemetry module.

 
\end{enumerate}

\underline{Multirotor Software}

> The autopilot flight computer uses open source software that the arudpilot website refers to as ‘firmware’. Because the board is an older version it actually does not have enough physical memory to fit the latest iterations of the firmware. As such we are using a legacy version of the firmware on our autopilot flight computer.

Reviewer: ```suggestion
The autopilot flight computer uses open source firmware from ArduPilot. The autopilot is an older version and has insufficient physical memory to fit the latest version of the firmware. As such we are using a legacy version of the firmware on our autopilot flight computer.
```

\underline{Ground station hardware}

The hardware for the ground station that is relevant to the autopilot is simply a computer and a telemetry system that will allow communication between the ground station and the multirotor.

\underline{Ground station software}

> The ground station uses open source software called APM planner 2.0. The software is downloaded onto a laptop and provides the display, means of controller, and means of interfacing with the autopilot, including uploading the firmware.

Reviewer: Convert to present tense

\subsection{Features list}

\begin{enumerate}

>     \item Real time monitoring of telemetry information: Data includes the altitude, position and orientation of the multirotor as well as real-time battery monitoring

Reviewer: - "real-time"?
- "telemetry information" is redundant, just use "telemetry"

```suggestion
    \item Real-time telemetry: altitude, position, orientation, and battery levels
```
